process_report/util.py

```diff
@@ -34,6 +34,28 @@ def load_institute_list():
     return institute_list
 
 
+def get_institute_mapping(institute_list: list):
+    institute_map = dict()
+    for institute_info in institute_list:
+        for domain in institute_info["domains"]:
+            institute_map[domain] = institute_info["display_name"]
+
+    return institute_map
+
+
+def get_institution_from_pi(institute_map, pi_uname):
```

Comment:
I have seen this method somewhere before

Reply:
I've added a commit to move the institution functions to `util.py`